lib/wpt-runner.js

```diff
@@ -116,10 +116,16 @@ function setupServer(testsPath, rootURL) {
 
 function runTest(url, setup, reporter) {
   return new Promise(resolve => {
+    const virtualConsole = new VirtualConsole()
+      .sendTo(console, { omitJSDOMErrors: true })
+      // log original error stack if available
+      // eslint-disable-next-line no-console
+      .on("jsdomError", e => console.error(e.detail.stack || e.detail));
```

Comment:
Hmm, the jsdom documentation seems to imply that `e.detail` may not exist? [From the README:](https://github.com/jsdom/jsdom/blob/13.0.0/README.md#virtual-consoles)
> If you're using `sendTo(c)` to send errors to `c`, by default it will call `c.error(errorStack[, errorDetail])` with information from `"jsdomError"` events.

If we are completely paranoid, I guess we could change it to:
```js
console.error((e.detail && e.detail.stack || e.detail) || e.stack || e)
```

Reply:
Yeah, that's correct, for example not-implemented errors do not have a `.detail`.

I can fix and push.